examples/06-verif-manual/vm-009-large_lateral_deflection_of_unequal_stiffness_springs.py

```diff
@@ -0,0 +1,409 @@
+   performed due to the fact that the resistance to the load is a function of
+   the deformed position. ``POST1`` is used to extract results from
+   the solution phase.
+
+"""
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+# Start MAPDL and import Numpy and Pandas libraries.
+
+# sphinx_gallery_thumbnail_path = '_static/vm9_setup.png'
+# sphinx_gallery_thumbnail_path = '_static/vm9_setup_1.png'
+# sphinx_gallery_thumbnail_path = '_static/vm9_setup_2.png'
+
+import numpy as np
+import pandas as pd
+
+from ansys.mapdl.core import launch_mapdl
+
+# Start MAPDL.
+mapdl = launch_mapdl()
+
+
+###############################################################################
+# Pre-Processing
+# ~~~~~~~~~~~~~~
+# Enter verification example mode and the pre-processing routine.
+
+mapdl.clear()
+mapdl.verify()
+_ = mapdl.prep7()
+
+
+###############################################################################
+# Parameterization
+# ~~~~~~~~~~~~~~~~
+# Parameterization block includes main constants as :
+#
+# * :math:`l = 10\,cm` - spring length.
+# * :math:`k_1 = 8\,N/cm` - stiffness of the 1st spring.
+# * :math:`k_2 = 1\,N/cm` - stiffness of the 2nd spring.
+# * :math:`m = 1` - mass.
+# * :math:`F = 5\sqrt[2]{2}\,N` - main load
+# * :math:`\alpha = 45\,º` - force angle
+# * :math:`c_x = \sqrt[2]{k_xm} = 1,41` - damping coefficient, x-direction.
+# * :math:`c_y = \sqrt[2]{k_ym} = 2.0` - damping coefficient, y-direction.
+
+# Main constants.
+length = 10
+k_spring1 = 8
+k_spring2 = 1
+c_damp_x = 1.41
+c_damp_y = 2.0
+mass = 1
+
+# Fx and Fy has been obtained by the projection F on the X and Y axes.
+f_x = 5
+f_y = 5
+
+
+###############################################################################
+# Define Element Type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element types.
+
+# Element type COMBIN14.
+mapdl.et(1, "COMBIN14")
+
+# Special Features are defined by keyoptions of the element COMBIN14.
+# KEYOPT(3)(2)
+# Degree-of-freedom selection for 2-D and 3-D behavior:
+# 2-D longitudinal spring-damper (2-D elements must lie in an X-Y plane)
+mapdl.keyopt(1, 3, 2)
+
+# Element type COMBIN40.
+mapdl.et(3, "COMBIN40")
+
+# Special Features are defined by keyoptions of the element COMBIN40.
+# KEYOPT(3)(1)
+# Element degrees of freedom:
+# UX (Displacement along nodal X axes)
+mapdl.keyopt(3, 3, 1)
+
+# KEYOPT(6)(2)
+# Mass location:
+# Mass at node J
+mapdl.keyopt(3, 6, 2)
+
+# Element type COMBIN40.
+mapdl.et(4, "COMBIN40")
+
+# Special Features are defined by keyoptions of the element COMBIN40.
+# KEYOPT(3)(2)
+# Element degrees of freedom:
+# UX (Displacement along nodal X axes)
+mapdl.keyopt(4, 3, 2)
+
+# KEYOPT(6)(2)
+# Mass location:
+# Mass at node J
+mapdl.keyopt(4, 6, 2)
+
+# Print the list of the elements and their attributes.
+print(mapdl.etlist())
+
+# Print the list of the elements and their attributes.
+print(mapdl.elist())
```

Comment:
```suggestion
```
You don't have still elements, so this prints nothing.

Reply:
Totally agree ! Thank you!